tokenserver/assignment/sqlnode/schemas.py

```diff
@@ -40,6 +40,8 @@ class _UsersBase(object):
     a uid, an allocated node, and last-seen generation and client-state values.
     Rows are timestamped for easy cleanup of old records.
     """
+    # Not sure why, but sqlalchemy + sqlite aren't noting this field as
+    # auto-incrementing anyomre.
```

Comment:
If we now believe autoincrement to be working as intended, let's delete this comment about it not working.